### ISSUE TITLE
run image as a non 1000 user prevent from editing files

### ISSUE THREAD

**Issue by zesk06** [nemanjan00/dev-environment#1]
TL;DR: Do you have any hint on using this technique if you user is not 1000?

Context: 
I am a docker-vim person like you, and i ended up with many plugins, and also wrote a Dockerfile to make my dev env 'portable' in a docker image.
But, I struggle with the uid defined in the Dockerfile, usually 1000 like yours.
My uid is not the same on every computer which prevent from mounting current PWD to a volume for edition within the container.
So I ended up adding a variable to the Dockerfile MUID, and creating as many docker image as I have different UID.
See [Dockerfile](https://gitlab.com/zesk06/dotfiles/-/blob/master/Dockerfile)

Do you have the same issue with UID?
Do you see any solution?

**Comment by nemanjan00** [comment]
You could make user dir more permissive, and do something like this:

```
docker run --user 1337:1000 -ti -eTERM=xterm-256color -eHOME=/work -v$(pwd):/work/project nemanjan00/dev zsh -ic "cd project ; tmux"
```
